app/Http/routes.php

```diff
@@ -197,6 +197,8 @@
 {
     Route::get('myresumes/{id}',['as' => 'cabinet.my_resumes' ,'uses' => 'cabinet\CabinetController@showMyResumes']);
     Route::get('myvacancies/{id}',['as' => 'cabinet.my_vacancies' ,'uses' => 'cabinet\CabinetController@showMyVacancies']);
+    Route::get('project/myaddvacancies',['as' => 'cabinet.my_addvacancies' ,'uses' => 'cabinet\CabinetController@showAddVacancies']);
+    Route::get('project/myaddvacancy/{id}',['as' => 'cabinet.my_addvacancy' ,'uses' => 'Vacancy\VacancyController@getVacancy']);
```

Comment:
Do we need this route?

Reply:
I think yes. Otherwise how to get the right json?